src/components/checkbox/checkbox.scss

```diff
@@ -1,39 +1,53 @@
-//$checkbox-width: 20px !default;
+//$checkbox-width: 18px !default;
 //$checkbox-height: $checkbox-width !default;
 //$checkbox-border-radius: 2px !default;
 //$checkbox-border-width: 2px !default;
 //
 // ^^ defined in _variables.scss
 //
-$checkbox-margin: 16px !default;
-$checkbox-text-margin: 10px !default;
-$checkbox-top: 12px !default;
+$checkbox-margin-end: 16px !default;
+$checkbox-text-margin-top: 10px !default;
+$container-checkbox-margin: 3px !default;
+
+$checkbox-min-height: 48px !default;
+$checkbox-min-height-dense: 36px !default;
+$checkbox-text-margin: 36px !default;
+
+// from input.scss
+$input-container-margin-top: 18px !default;
+$input-container-padding-top: 2px !default;
+$input-padding-top: 2px !default;
+$input-padding-bottom: 1px !default;
+$input-border: 1px !default;
+
+$md-inline-alignment: $input-container-margin-top + $input-container-padding-top
+                    + $input-padding-top + $input-padding-bottom + $input-border
+                    - $checkbox-text-margin-top !default;
 
 .md-inline-form {
   md-checkbox {
-    margin: 19px 0 18px;
+    margin-top: $md-inline-alignment;
+    margin-bottom: auto;
   }
 }
 
 md-checkbox {
   box-sizing: border-box;
   display: inline-block;
-  margin-bottom: $checkbox-margin;
   white-space: nowrap;
   cursor: pointer;
   outline: none;
   user-select: none;
   position: relative;
   min-width: $checkbox-width;
-  min-height: $checkbox-width;
```

Comment:
@Splaktar Comparing against my personal fork, from 2016, I added this line:

`min-height: $checkbox-width * 2;`

https://github.com/clshortfuse/material/commit/be11f583a3d425cfef4969b933aaa8e51e8e358a#diff-9d193e89cd7efd0fa1252d174ec1ffbdR34

I can't remember why, but surely it must been for a reason. Then again, I did change a lot of other things, so it may not be need in the context of a single commit. It's something to keep in mind. It may be `36px` would be a reasonable minimum touch target.

The other stuff in the commit for `md-checkbox` is related to horizontal spacing (`md-spacing` attribute), since dense only applies vertically.

Reply:
Thank you for taking a look at this after so long!

`$checkbox-min-height` is `48px` and `$checkbox-min-height` is `36px`.

I think that the current code (`@include dense(min-height, $checkbox-min-height, $checkbox-min-height-dense);`) is correct and just an updated version of what you had locally.